pkg/util/i18n/i18n.go

```diff
@@ -53,10 +53,6 @@ func loadSystemLanguage() string {
 		return "default"
 	}
 	pieces := strings.Split(langStr, ".")
-	if len(pieces) == 0 {
```

Comment:
Oh, that is true. And I have updated the PR and PTAL. Thanks a lot. @cblecker @brendandburns